### PR TITLE
Improve `from_pretrained` as #13

### PR DESCRIPTION
**Pull request by AaronHeee**
What we have done:

1. rewrite `from_pretrained` to support NLTK pretokenizer;
2. formatting.

Todo:

1. Add docstring.

### CODE REVIEW

File: src/recwizard/modules/kbrd/tokenizer_kbrd_gen.py [hunk 1]
```diff
@@ -1,19 +1,62 @@
...
+        self.entity_tokenizer = entity_tokenizer
+        tokenizers = [self.word_tokenizer, self.entity_tokenizer]
+
+        if word_tokenizer is None or entity_tokenizer is None:
+            raise ValueError("word_tokenizer and entity_tokenizer should not be None.")
+
+        super().__init__(tokenizers=tokenizers, **kwargs)
+
+    @classmethod
+    def from_pretrained(cls, pretrained_model_name_or_path, *args, **kwargs):
```
Comment (Kevin99z) [inline src/recwizard/modules/kbrd/tokenizer_kbrd_gen.py:31]: could use get_init_kwargs to save the init word list to the hf hub.

Comment (AaronHeee) [reply src/recwizard/modules/kbrd/tokenizer_kbrd_gen.py:31]: Thank you, that would be much cleaner. I managed to get it working today and will push it once I finish the docstrings.

Comment (AaronHeee) [reply src/recwizard/modules/kbrd/tokenizer_kbrd_gen.py]: Hey @Kevin99z, I use `get_init_kwargs` now and finish the docstring part for our code. Please review it when you have time, thanks!

Comment (Kevin99z) [reply src/recwizard/modules/kbrd/tokenizer_kbrd_gen.py]: LGTM

**Pull request merged**